### ISSUE TITLE
about “gradient clipping” in optimizers

### ISSUE THREAD

**Issue by boozyguo** [fchollet/keras#3414]
Is Keras able to support "clipping gradient" such as Caffe?  In order to avoid the emergence of "Nan" in training?

**Comment by paul-mannino** [comment]
yes, optimizers can take a 'clipvalue' argument, e.g.
`optimizer = RMSprop(lr=0.01, clipvalue=100)`

**Comment by boozyguo** [comment]
hi, paul-mannino.  Thank you for reply.  
I also try add maxvalue to activation function, e.g.
relu(x, maxvalue=1.0)
But there are "Nan" also. 
I will try to add "clipvalue" in optimizer.  And also I will try to set the initial value for bias and weights.
Thanks a lot!
